applications_tests/lethe-fluid-particles/adaptive_sparse_contacts.prm

```diff
@@ -0,0 +1,215 @@
+# Listing of Parameters
+#----------------------
+
+set dimension = 3
+
+# --------------------------------------------------
+# Simulation and IO Control
+#---------------------------------------------------
+
+subsection simulation control
+  set method               = bdf1
+  set number mesh adapt    = 0
```

Comment:
Agreed

Reply:
I agree, but I don't know all the default parameters, especially for the cfd part. 
I will try to remove the ones I find quickly in the parameter.cc, but since most of the application tests do have default parameters, it is more a clean up thing in my opinion.

Reply:
For all the tests in Lethe